integration-tests/support/commands/hooks.ts

```diff
@@ -0,0 +1,6 @@
+import { Login } from "../../utils/Login";
+
+before(() => {
+    //Login to Rad Hat SSO
+    Login.login(Cypress.env('USERNAME') ,Cypress.env('PASSWORD'));
+  });
```

Comment:
Shouldn't this also use prettier to format it properly? Tab spacing is a bit off here and it should have a new line at the end.

Reply:
Done, Thanks :)